net.sf.eclipsecs.core/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-ClassPath: .,
  lib/dom4j-2.1.3.jar,
  lib/snakeyaml-1.26.jar,
```

Comment:
Completely unrelated to this PR, but I sent a PR to another repo to upgrade this library because of security concerns.
https://github.com/checkstyle/sonar-checkstyle/pull/450#issue-1448041887

Reply:
Here it is: https://github.com/checkstyle/eclipse-cs/pull/402